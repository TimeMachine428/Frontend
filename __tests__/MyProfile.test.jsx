import React from 'react';
import TermsConditions from '../src/js/pages/TermsConditions.jsx';
import renderer from 'react-test-renderer';

test('TermsConditions page information correct',() =>{
	const component = renderer.create(
		<TermsConditions />,
	);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
})