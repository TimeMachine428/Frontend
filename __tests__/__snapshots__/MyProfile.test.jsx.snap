// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TermsConditions page information correct 1`] = `
<div>
  <h1>
    Project Time Machine Terms of Use
  </h1>
  <p>
    A. INTRODUCTION TO THE SERVICE
    <br />
    Our services are all interactions on this web domain. Our service has the Goal of helping 
    <br />
    users learn and improve in different aspects of coding. These terms govern what is posted, 
    <br />
    what is read and all interactions between users. To Use our services, an internet 
    <br />
    connection is required. This may affect our Services' performance.
    <br />
    B. BY USING OUR SERVICE
    <br />
    Your confidential information will be protected. This includes passwords, billing information 
    <br />
    and other personal details. All transactions are final unless extenuating circumstances. All 
    <br />
    users of our Service must follow the USER GUIDELINES.
    <br />
    C. USER GUIDELINES
    <br />
    Users must interact with respect when using our Service. it is strictly prohibited to: 
    <br />
    - post any materials that you do not have permission, right or license to use;
    <br />
    - post objectionable, offensive, unlawful, deceptive or harmful content;
    <br />
    - post personal, private or confidential information belonging to others;
    <br />
    - request personal information from a minor;
    <br />
    - pronounce 'gif' as 'jif';
    <br />
    - impersonate or misrepresent your affiliation with another person, or entity;
    <br />
    - post or transmit spam, including but not limited to unsolicited or unauthorized advertising, 
    <br />
    promotional materials, or informational announcements;
    <br />
    - plan or engage in any illegal, fraudulent, or manipulative activity;
    <br />
    - say PHP is your favorite programming language.
    <br />
  </p>
</div>
`;
